client/data/support-article-alternates/use-support-article-alternates-query.js

```diff
@@ -0,0 +1,24 @@
+import { isDefaultLocale } from '@automattic/i18n-utils';
+import { useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+import wp from 'calypso/lib/wp';
+import getCurrentLocaleSlug from 'calypso/state/selectors/get-current-locale-slug';
+
+function useSupportArticleAlternatesQuery( blogId, postId, queryOptions = {} ) {
+	const locale = useSelector( getCurrentLocaleSlug );
+
+	return useQuery(
+		[ 'support-article-alternates', blogId, postId ],
+		() => wp.req.get( `/support/alternates/${ blogId }/posts/${ postId }` ),
+		{
+			...queryOptions,
+			enabled: ! isDefaultLocale( locale ) && !! ( blogId && postId ),
+			refetchOnWindowFocus: false,
```

Comment:
Would it make sense to add `refetchOnMount: false` as well? Since it's being used within a dialog, it'd always mount/unmount triggering the refetch, which seems unnecessary in this case.

Reply:
Yup, I think it makes sense. Disabled mount refetching in https://github.com/Automattic/wp-calypso/pull/62161/commits/37174696c5b79fd3db9386cbf9cc131382670977.